src/WebSite Ui Components/OfflinePage/OfflinePage.js

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useState } from "react";
+import "!style-loader!css-loader!./OfflinePage.css";
+import { PUBLIC_ASSETS_PATH } from "../../Utils/Constants";
+import PlanetDefense from "./PlanetDefense";
+import OrbitalsDraw from "./OrbitalsDraw";
+
+function OfflinePage() {
+  const [offline, setOffline] = useState(false);
+  const [modeOne, setModeOne] = useState(false);
+  const [modeTwo, setModeTwo] = useState(false);
+
+  useEffect(() => {
+    if (!navigator.onLine) setOffline(true);
+  }, []);
+
+  const PlanetGame = () => {
+    setModeOne(true);
+  };
+  const DrawOrbitals = () => {
+    setModeTwo(true);
+  };
+  return (
+    <>
+      {offline && (
+        <div>
+          {modeTwo ? <OrbitalsDraw /> : null}
```

Comment:
rename it ... use single state and chagne its value to GAME_TYPE (PLANET_GAME/CIRCLE_GAME)